tests/python/relay/aot/test_crt_aot_usmp.py

```diff
@@ -18,16 +18,23 @@
 
 from collections import OrderedDict
 import re
-
+import textwrap
```

Comment:
why is this needed ?

Reply:
textwrap.dedent used to strip common leading whitespaces.